frame/contracts/src/wasm/runtime.rs

```diff
@@ -2076,4 +2086,20 @@ define_env!(Env, <E: Ext>,
 			Err(_) => Ok(ReturnCode::EcdsaRecoverFailed),
 		}
 	},
+
+	// Returns then number of times currently executing contract exists on the call stack in addition
+	// to the calling instance. A value of 0 means no reentrancy.
+	[__unstable__] seal_reentrant_count(ctx) -> u32 => {
+		ctx.charge_gas(RuntimeCosts::ReentrantCount)?;
+		Ok(ctx.ext.reentrant_count() as u32)
+	},
+
+	// Returns the number of times specified contract exists on the call stack. Delegated calls are
```

Comment:
please make the comment look like the other ones, with the `Parameters` and `Return Value` sections